docs/developers/ESLint_Plugins.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: eslint-plugins
+sidebar_label: Building ESLint Plugins
+title: ESLint Plugins
+---
+
+:::important
+This page describes how to write your own custom ESLint plugins using typescript-eslint.
+You should be familiar with [ESLint's plugins guide](https://eslint.org/docs/latest/extend/plugins) and [typescript-eslint Custom Rules](./Custom_Rules.mdx) before writing custom plugins.
+:::
+
+Custom plugins that support TypeScript code and typed linting look very similar to any other ESLint plugin.
+Follow the same general steps as [ESLint's plugins guide > _Creating a plugin_](https://eslint.org/docs/latest/extend/plugins#creating-a-plugin) to set up your plugin.
+The required differences are noted on this page.
+
+:::tip
+See [**`eslint-plugin-example-typed-linting`**](https://github.com/typescript-eslint/examples/tree/39a6f7bb310645c88382a25aa8bed719ccd61a0c/packages/eslint-plugin-example-typed-linting) for an example plugin that supports typed linting.
```

Comment:
Use `main` instead of a hash; it's better for the link to be broken (and fixed) rather than outdated by default.

Reply:
I'll go ahead and merge https://github.com/typescript-eslint/examples/pull/13 to get it on that repo's `main`. 👍